### PR TITLE
Privacy-Preserving Analytics Settings Polishing (DEV)

### PR DESCRIPTION
**Pull request by axelherbstreith**
This PR adapts some smaller thing in the privacy-preserving analytics UI.

Following is currently including:
- Adaption of settings icons
- Added link to _ppa_more_information_fragment_
- Removed unused margin below the scroll view
- Used predefined dimes for margins in the layouts
- Aligned fonts of the bullet points in "Your Consent" container with the other parts of the fragment
- Added clickListener to analytics row that contains the switch (not needed to directly click on the switch anymore)
- Aligned font of the Privacy Declaration Box with other parts of the screen

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/analytics/SettingsPrivacyPreservingAnalyticsFragment.kt [hunk 1]
```diff
@@ -33,17 +33,29 @@ class SettingsPrivacyPreservingAnalyticsFragment :
 
         binding.apply {
 
+            // Notifications about risk status
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/analytics/SettingsPrivacyPreservingAnalyticsFragment.kt:36]: Copy paste? :D

**Review by harambasicluka — COMMENTED** [review]
LGTM, will test it on device now.

**Review by harambasicluka — APPROVED** [review]
Looks nice in dark and light mode!
- Pixel 3

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/analytics/SettingsPrivacyPreservingAnalyticsFragment.kt (hunk 1 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/analytics/SettingsPrivacyPreservingAnalyticsFragment.kt:36]: maybe :D

**Review by AlexanderAlferov — APPROVED** [review]

**Pull request merged**